vmm/src/cpu.rs

```diff
@@ -106,6 +110,22 @@ pub enum Error {
 
     /// Failed scheduling the thread on the expected CPU set.
     ScheduleCpuSet,
+
+    #[cfg(all(target_arch = "x86_64", feature = "gdb"))]
+    /// Error on debug related CPU ops.
+    CpuDebug(hypervisor::HypervisorCpuError),
+
+    #[cfg(all(target_arch = "x86_64", feature = "gdb"))]
+    /// Error on debug related VM ops.
+    VmDebug(hypervisor::HypervisorVmError),
+
+    #[cfg(all(target_arch = "x86_64", feature = "gdb"))]
+    /// Failed to translate guest virtual address.
+    TranslatingVirtAddr,
```

Comment:
There is a `hypervisor::TranslateVirtualAddress` already. You can perhaps wrap that in here.

Reply:
8226d43 adds the suggested changes.